pkg/gen/input/workflows/internal/file/create_release.go

```diff
@@ -217,8 +217,14 @@ jobs:
       - name: Create long-lived release branch
         run: |
           current_version="${{ needs.gather_facts.outputs.version }}"
-          parent_version="$(git describe --tags --abbrev=0 HEAD^)"
+          parent_version="$(git describe --tags --abbrev=0 HEAD^ || true)"
           parent_version="${parent_version#v}" # Strip "v" prefix.
+
+          if [[ "$parent_version" = "" ]] ; then
```

Comment:
Feels like we should use `-z`.

Reply:
Yeah that's probably better. Changed this